float64 M1_current
float64 M2_current
float64 Voltage